tools/config/seed.config.ts

```diff
@@ -1,5 +1,6 @@
 import { join } from 'path';
 import { argv } from 'yargs';
+import * as slash from 'slash';
```

Comment:
Can you please order the imports alphabetically to be consistent with the [Styleguide](https://angular.io/styleguide#!#03-06)